### PR TITLE
22843-Cannot-select-all-in-windows

### PR DESCRIPTION
**Pull request by pavel-krivanek**
do an image patch for Windows VM inconsistency with other platforms. This OS returns keyCharacters as uppercase

### CODE REVIEW

File: src/Morphic-Core/KeyboardEvent.class.st [hunk 1]
```diff
@@ -107,11 +107,14 @@ KeyboardEvent >> keyValue [
...
 { #category : #keymapping }
 KeyboardEvent >> modifiedCharacter [
-    self flag: #hack.
-    "Hack me.  When Ctrl is pressed, the key ascii value is not right and we have to do something ugly"
-    ^(self hasSpecialCTRLKeyValue and: [ (#(MacOSX Windows) includes: Smalltalk os current platformFamily) ])
-        ifTrue: [ (self keyValue + $a asciiValue - 1) asCharacter ]    
-        ifFalse: [ self keyCharacter ]
+	self flag: #hack.
+	"Hack me.  When Ctrl is pressed, the key ascii value is not right and we have to do something ugly"
+	^(self hasSpecialCTRLKeyValue and: [ (#(MacOSX Windows) includes: Smalltalk os current platformFamily) ])
```
Comment (estebanlm) [inline src/Morphic-Core/KeyboardEvent.class.st]: `Smalltalk os platformFamily` is enough here. Why the "current" ?

Comment (estebanlm) [reply src/Morphic-Core/KeyboardEvent.class.st]: meh, I see it was like that before.
still no need :)

Comment (jecisc) [reply src/Morphic-Core/KeyboardEvent.class.st]: @estebanlm I don't know if you know, but now you can suggest one line changes in reviews to make it easier to contribute.

firefox_2019-01-04_10-18-31

You can use this icon.

Comment (jecisc) [inline src/Morphic-Core/KeyboardEvent.class.st]:
```suggestion
	^(self hasSpecialCTRLKeyValue and: [ (#(MacOSX Windows) includes: Smalltalk os platformFamily) ])
```

**Review by jecisc — CHANGES_REQUESTED** [review]

**Comment by jecisc** [comment]
Except the little cleanup requested, the PR works for me on Windows 7.

**Review by jecisc — APPROVED** [review]

**Comment by jecisc** [comment]
Test failure not related.

**Pull request merged**